e2e/specs/st_main_menu.spec.js

```diff
@@ -23,22 +23,49 @@ describe("main menu", () => {
     cy.get("[data-testid='stDecoration']").invoke("css", "display", "none");
   });
 
-  it("displays menu dropdown", () => {
+  it("displays light main menu and about section properly", () => {
     cy.get("[data-testid='stConnectionStatus']").should("not.exist");
 
     // Main menu renders visually as we expect
     cy.get("#MainMenu > button").click();
+
     // Cypress cuts the popover off due to the transform property, so we move
     // the main menu to a location to show it clearly for snapshots.
     cy.get('[data-testid="main-menu-popover"]').invoke(
       "attr",
       "style",
       "transform: translate3d(20px, 20px, 0px)"
     );
-    cy.get('[data-testid="main-menu-list"]').matchImageSnapshot("main_menu");
+
+    cy.get('[data-testid="main-menu-list"]')
+      .eq(0)
+      .matchImageSnapshot("main_menu");
+
+    cy.get('[data-testid="main-menu-list"]')
+      .eq(1)
+      .matchImageSnapshot("dev_main_menu");
 
     // Not possible to test the urls in the menu as they are hidden behind
     // the click handler of the button
     // https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/testing-dom__tab-handling-links/cypress/integration/tab_handling_anchor_links_spec.js
+
+    // need to add testing for about section
+  });
+
+  it("displays dark main menu and about section properly", () => {
+    cy.get("#MainMenu > button").click();
```

Comment:
is this first click necessary? I think it can be removed since `cy.changeTheme` does everything for you

Reply:
It's necessary because the last test is still on main menu so this click closes it. Maybe I can put it in the before test?

Reply:
Ah okay, I think leaving a comment mentioning why it's needed should be okay